### DISCUSSION TITLE
[mega thread]: Documentation Site feedback

### DISCUSSION THREAD

**Discussion by joshuaellis** [pmndrs/react-spring#1931] [New Docs site feedback]
Link to beta – https://beta.react-spring.dev/

This thread should focus on:
* Any bugs you find
* Potential improvements to the site (although these may not be actioned so please be aware)
* Content changes // improvements // suggestions etc.

Thank you in advance for all your time and effort to contributing. If you're happy with the direction the docs are going & want to contribute further, consider sponsoring (or getting your company to sponsor) react-spring on [open collective](https://opencollective.com/react-spring) 🙇🏻‍♀️

**Comment by iamzapata** [comment]
Love the new look! ❤️

**Comment by dylanbr0wn** [comment]
Loving the new UI and the content improvements! 

It appears the bookmark links at the top of the pages are a little buggy. On my Windows machine in Chrome, the links mostly require 2 clicks each before they will navigate. On Firefox, these links often didn't work at all for me and just bounced me all over the page.

In Firefox:

https://user-images.githubusercontent.com/40218657/178317618-de587dde-81de-489d-97af-ffa0a5441055.mp4

**Reply by joshuaellis** [reply]
thanks for the report!

**Comment by J4v4Scr1pt** [comment]
The new look are awesome 🥳!
You wrote on the site "_why not contribute with your own example_".
I did a draggble button this summer with the help of you and if you would like to use it as a example feel free to do so 😄:
https://codesandbox.io/s/draggableelement-uyb9ho

**Reply by joshuaellis** [reply]
This looks great, a lot of potential, I think it would need some tweaking before we include it as an example, but it's pretty much there 👍🏼

**Reply by J4v4Scr1pt** [reply]
Thx, yee it's a bit basic atm 😊. If there is anything you need from me plz let me know otherwise you are free to do whatever you want with it 👍.

**Comment by holtzy** [comment]
👋 Small typo in front page:

`Repsond to events`

Thanks so much for your work!! 🙏

**Comment by anshullaikar** [comment]
![image](https://user-images.githubusercontent.com/22425559/206363323-e72d0dfa-ba27-44e5-8a79-9a4da2167391.png)
Hi, the website seems to be broken, not sure if this is only happening for me.

**Comment by Caveman-Dan** [comment]
Hi,

All links to examples on the website seem to be broken: [see here](https://www.react-spring.dev/examples)

Thank you

**Reply by joshuaellis** [reply]
Thanks. There's a fix ready to be deployed I just can't get into Vercel at the moment
